packages/stress/src/mode/chat/schema/commands/expectChannelMessageCommand.ts

```diff
@@ -0,0 +1,121 @@
+import { z } from 'zod'
+import { baseCommand } from './baseCommand'
+import { StressClient } from '../../../../utils/stressClient'
+import { ChatConfig } from '../../../common/types'
+import { channelMessagePostWhere } from '../../../../utils/timeline'
+import { Stream } from '@river-build/sdk'
+import { Logger } from 'pino'
+import { CodeException } from '@river-build/dlog'
+
+const expectedMessage = z.object({
+    content: z.string(),
+    sender: z.string().optional(),
+})
+
+const paramsSchema = z.object({
+    channelId: z.string(),
+    timeoutMs: z.number().nonnegative().optional(),
+    messages: z.array(expectedMessage),
+})
+
+export const expectChannelMessageCommand = baseCommand.extend({
+    name: z.literal('expectChannelMessage'),
+    params: paramsSchema,
+})
+
+export type ExpectChannelMessageParams = z.infer<typeof paramsSchema>
+export type ExpectChannelMessageCommand = z.infer<typeof expectChannelMessageCommand>
+
+function escapeMessage(template: string, client: StressClient, cfg: ChatConfig): string {
```

Comment:
send and expect commands allow for escapes to make observing stress tests and detecting messages easier.